packages/radix-ui-themes/src/helpers/get-root.tsx

```diff
@@ -8,7 +8,7 @@ interface GetChildrenArgs<T extends React.ElementType> {
 }
 
 export const getRoot = <
-  T extends React.ElementType,
+  T extends React.ComponentType<any> | keyof JSX.IntrinsicElements,
```

Comment:
React.ElementType is different between workos and this repo because @types/react versions diverge, which our build toolchain complained about. This yields a similar result, but without the potential clashes of the type versions.